bliss/encoder.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         self,
         architecture: DictConfig,
         bands: list,
+        z_score: bool,
```

Comment:
It would probably be cleaner to include this in the `input_transform_params` dict

Reply:
I believe it is, I just forgot to remove this before committing. Will fix!